lib/cadet/accounts/notification.ex

```diff
@@ -79,12 +91,14 @@ defmodule Cadet.Accounts.Notification do
   @spec acknowledge(:integer, %User{}) :: {:ok} | {:error, Ecto.Changeset.t()}
   def acknowledge(notification_id, user = %User{}) do
     IO.puts("Acknowledge called")
-    IO.puts("with notification id: ")
-    IO.inspect(notification_id)
-    IO.puts("with user: ")
-    IO.inspect(user)
-
-    # Test
-    {:ok, nil}
+    IO.inspect(notification_id, label: "with notification_id")
+    IO.inspect(user, label: "with user")
+
+    from(n in Cadet.Accounts.Notification,
+      where: n.user_id == ^user.id and n.id == ^notification_id
+    )
+    |> Repo.one!()
```

Comment:
https://github.com/geshuming/cadet/blob/ab3a3c82407717cd3f034849b6edf7319532010b/lib/cadet/accounts/notification.ex#L97-L100

This segment can be replaced by
```suggestion
Repo.get_by(Notification, id: notification_id, user_id: user.id)
```